src/dummy/codes.json

```diff
@@ -3,70 +3,153 @@
         "id": 1,
         "words": [
             {"id": 1, "word": " "},
-            {"id": 2, "word": "prublic"},
+            {"id": 2, "word": "public"},
```

Comment:
Yes. The Json simulates our data from the backend. In Future the lecturer inputs the code in a text  in the lecturer-interface field an it will be parsed automatically and saved in the bugfinder-backend